galaxy.yml

```diff
@@ -1,13 +1,41 @@
----
```

Comment:
yamlint is going to warn that this `---` is missing.

./galaxy.yml
  6:1       warning  missing document start "---"  (document-start)
  35:2      warning  missing starting space in comment  (comments)